addons/web/static/src/search/search_model.js

```diff
@@ -774,10 +774,9 @@ export class SearchModel extends EventBus {
     getSearchItems(predicate) {
         const searchItems = [];
         Object.values(this.searchItems).forEach((searchItem) => {
-            if (
-                (!("invisible" in searchItem) || !searchItem.invisible) &&
-                (!predicate || predicate(searchItem))
-            ) {
+            const isInvisible =
+                "invisible" in searchItem && evaluateExpr(searchItem.invisible, this.globalContext);
```

Comment:
searchItem.invisible is not always a string! see splitAndAddDomain

Reply:
we should add a test that involves the domain selector dialog, the filter menu and the groupby menu maybe.